src/main/java/com/amazon/opendistroforelasticsearch/sql/query/planner/converter/SQLToOperatorConverter.java

```diff
@@ -76,6 +82,19 @@ public boolean visit(MySqlSelectQueryBlock query) {
         return aggregationParser.getColumnNodes();
     }
 
+    public List<String> extractSelectFunctionNames(List<SQLSelectItem> selectItems) {
+        List<String> methodNames = new ArrayList<>();
+        for (SQLSelectItem selectItem: selectItems){
+            SQLExpr selectItemExpr = selectItem.getExpr();
+            if (selectItemExpr instanceof SQLMethodInvokeExpr) {
+                methodNames.add(((SQLMethodInvokeExpr) selectItemExpr).getMethodName());
+            } else {
+                methodNames.add(null);
```

Comment:
Avoid to use null value, Consider to use Optional if it is necessary.

Reply:
Will take another look.